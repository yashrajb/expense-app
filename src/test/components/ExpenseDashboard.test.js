import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboard from '../../components/ExpenseDashboardPage.js';

test('should render dashboard page',() => {

    const result = shallow(<ExpenseDashboard />);
    expect(result).toMatchSnapshot();

});