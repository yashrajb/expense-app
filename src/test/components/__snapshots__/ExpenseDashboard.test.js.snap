// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard page 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
